src/components/Games/GameEditor/GameEditor.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react"
+import { useCallback, useEffect, useMemo, useState } from "react"
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
+import { Typography } from "decentraland-ui2"
+import { gameApi } from "../../../api/gameApi"
+import { GameRequest } from "../../../types"
+import { compareDataChanged } from "../../../utils/compareDataChanged"
+import { SaveButton } from "../../SaveButton/SaveButton"
+import { TextFieldStyled } from "../../TextFieldStyled/TextFieldStyled"
+import { GameEditorProps } from "./GameEditor.types"
+import { Container } from "./GameEditor.styled"
+
+const EMPTY_GAME = {
+  name: "",
+  x: 0,
+  y: 0,
+}
+
+const GameEditor = React.memo(({ gameData, onUpdate }: GameEditorProps) => {
+  const create = !gameData?.id
+  const initData: GameRequest = create
+    ? EMPTY_GAME
+    : {
+        name: gameData.name,
+        x: parseInt(gameData.parcel.split(",")[0]),
+        y: parseInt(gameData.parcel.split(",")[1]),
+      }
+
+  const [data, setData] = useState<GameRequest>(initData)
+  const [dataChanged, setDataChanged] = useState<Record<string, boolean>>({})
+
+  useEffect(() => {
+    setDataChanged(compareDataChanged(initData, data))
+  }, [initData, data])
+
+  const l = useFormatMessage()
+
+  const saveClickHandler = useCallback(async () => {
+    if (!(data.name && data.x && data.y)) return
+
+    if (create) {
+      await gameApi.createGame(data)
+    } else {
+      await gameApi.updateGame(gameData.id, data)
+    }
+
+    onUpdate && onUpdate()
+  }, [create, gameData?.id, data.name, data.x, data.y, onUpdate])
+
+  const hasNewChanges = useMemo(() => {
+    return Object.values(dataChanged).some((value) => value)
```

Comment:
I think you can do this here

```suggestion
    return Object.values(dataChanged).some(boolean)
```